src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -41,7 +43,13 @@ TableTabLabel.propTypes = {
 
 const message = "All numeric figures in this table are calculated based on the set of TAS owned by each agency, as opposed to the set of TAS that the agency directly reported to USAspending.gov. In the vast majority of cases, these are exactly the same (upwards of 95% of TAS—with these TAS representing over 99% of spending—are submitted and owned by the same agency). This display decision is consistent with our practice throughout the website of grouping TAS by the owning agency rather than the reporting agency. While reporting agencies are not identified in this table, they are available in the Custom Account Download in the reporting_agency_name field.";
 
-const AboutTheDataPage = () => {
+const AboutTheDataPage = ({
```

Comment:
Are we handling if a user navigates to `/about-the-data/agencies` do we default to the latest year/period?

Reply:
We are not. There shouldn't really be a need to do that, at least not in the scope of this ticket.